tests/testDipoleFitter.py

```diff
@@ -0,0 +1,407 @@
+from __future__ import absolute_import, division, print_function
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+import unittest
+
+from numpy import (random as np_random,
+                   array as np_array,
+                   mgrid as np_mgrid)
+
+import lsst.utils.tests as lsst_tests
+from lsst.afw.geom import (Box2I, Point2I, Point2D)
+from lsst.meas.algorithms import (SourceDetectionConfig, SourceDetectionTask)
+from lsst.afw.table import (SourceTable, SourceCatalog)
+from lsst.ip.diffim.dipoleFitTask import (DipoleFitAlgorithm, DipoleFitConfig, DipoleFitTask, DipolePlotUtils)
+from lsst.meas.base import SingleFrameMeasurementConfig
+
+# Export DipoleTestUtils to expose fake image generating funcs
+__all__ = ("DipoleTestUtils")
```

Comment:
Yes they should. I don't see an issue with having utility code in the main package. The tests should be tests.